client/config/toml.go

```diff
@@ -26,6 +26,8 @@ output = "{{ .Output }}"
 node = "{{ .Node }}"
 # Transaction broadcasting mode (sync|async|block)
 broadcast-mode = "{{ .BroadcastMode }}"
+# Map of denom aliases
+aliases = { uatom = "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2", uakt = "ibc/1480B8FD20AD5FCAE81EA87584D269547DD4D436843C1D20F15E00EB64743EF4", ujuno = "ibc/46B44899322F3CD854D2D46DEEF881958467CDD4B3B10086DA49296BBED94BED" }
```

Comment:
Can this be specified with multiple lines?

Or perhaps read more directly from a toml?

Reply:
yeah I also wanted to do that, but was getting an error